src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-database-page.mdx

```diff
@@ -10,7 +10,7 @@ metaDescription: Here are some tips for understanding the OpenTelemetry database
 After you've sent us your OpenTelemetry data and opened your service (entity) in the UI, click **Databases** to learn about the performance of the database queries your service is making. You can see which database calls (e.g., `SELECT` operation on the `ads` table) are taking the most time in total or on average, or have the most throughput. Then you can investigate performance of an individual call and find traces that involve it.
 
 ## Required attributes
-With OpenTelemetry, we rely on [`SpanKind`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#spankind) and the presence of the `db.system` attribute to identify spans representing client usage of a database.
+With OpenTelemetry, we rely on [`SpanKind`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#spankind) and the presence of the [`db.system`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md) attribute to identify spans representing client usage of a database.
```

Comment:
not sure about the wording on "client usage of a database." Do you think it might read better if it was made more personal to match the tone preceding paragraphs?